documentstore_migracao/processing/inserting.py

```diff
@@ -177,7 +219,10 @@ def register_documents_in_documents_bundle(
         data = documents_bundle["data"]
         items = documents_bundle["items"]
         try:
-            documents_bundle = get_documents_bundle(session_db, data)
+            if key.endswith("aop"):
+                documents_bundle = get_or_create_aop_bundle(session_db, data)
```

Comment:
key que termina com aop é um detalhe de implementação da classe que agrupa os documentos nos respectivos documents bundles. OK deixar assim se tiver um test que certifique que o KEY sempre terminará em aop se o bundle for de aop. Não sei se há este teste.

Reply:
@robertatakenaka, aqui fiquei em dúvida qual abordagem tomar. Após a discussão que tivemos, já seria ideal colocar aqui a verificação de inexistência de ano, volume e número?